variants/heltec_wireless_paper/variant.h

```diff
@@ -1,47 +1,41 @@
 #define LED_PIN 18
+#define BUTTON_PIN 0
 
-// Enable bus for external periherals
+// I2C
 #define I2C_SDA SDA
 #define I2C_SCL SCL
 
+// Display (E-Ink)
 #define USE_EINK
-
-/*
- * eink display pins
- */
 #define PIN_EINK_CS 4
 #define PIN_EINK_BUSY 7
 #define PIN_EINK_DC 5
 #define PIN_EINK_RES 6
 #define PIN_EINK_SCLK 3
 #define PIN_EINK_MOSI 2
 
-/*
- * SPI interfaces
- */
+// SPI
 #define SPI_INTERFACES_COUNT 2
+#define PIN_SPI_MISO 10 // MISO
+#define PIN_SPI_MOSI 11 // MOSI
+#define PIN_SPI_SCK 9   // SCK
 
-#define PIN_SPI_MISO 10 // MISO      P0.17
-#define PIN_SPI_MOSI 11 // MOSI      P0.15
-#define PIN_SPI_SCK 9   // SCK       P0.13
-
-#define VEXT_ENABLE 45 // active low, powers the oled display and the lora antenna boost
-#define BUTTON_PIN 0
-
+// Power
+#define VEXT_ENABLE 45 // Active low, powers the E-Ink display
 #define ADC_CTRL 19
 #define BATTERY_PIN 20
 #define ADC_CHANNEL ADC2_GPIO20_CHANNEL
 #define ADC_MULTIPLIER 2                // Voltage divider is roughly 1:1
 #define BAT_MEASURE_ADC_UNIT 2          // Use ADC2
-#define ADC_ATTENUATION ADC_ATTEN_DB_11 // Voltage divider output is quite high
+#define ADC_ATTENUATION ADC_ATTEN_DB_12 // Voltage divider output is quite high
```

Comment:
`ADC_ATTEN_DB_11` is now deprecated; identical to `ADC_ATTEN_DB_12`. This change removes compiler warning.